source/SharedCrtResourceManager.h

```diff
@@ -44,14 +44,18 @@ namespace Aws
                 aws_mem_trace_level memTraceLevel{AWS_MEMTRACE_NONE};
                 std::vector<Feature *> *features;
 
-                bool locateCredentials(const PlainConfig &config);
-
                 bool setupLogging(const PlainConfig &config);
 
                 int buildClient(const PlainConfig &config);
 
                 void initializeAllocator(const PlainConfig &config);
 
+              protected:
+                /**
+                 * inheritable for testing
+                 */
+                bool locateCredentials(const PlainConfig &config);
```

Comment:
If we are removing this method, then why are we not removing the code written in `source/SharedCrtResourceManager.cpp` file?

Reply:
This is not being removed.